easy_phi/app.py

```diff
@@ -451,7 +494,7 @@ def get(self):
 
 # URL schemas to RequestHandler classes mapping
 application = tornado.web.Application([
-    (r"/", IndexPageHandler,),
+    (r"/", IndexPageHandler, None, 'home'),
```

Comment:
@mansuleman these additions are actually names for URL patterns. They are used to refer to specific functions by these names, e.g. when we test this handler we can call use app.reverse_url('home') instead of hardcoded url